packages/smart-contracts/src/contracts/ChainlinkConversionPath.sol

```diff
@@ -1,8 +1,7 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >=0.4.25 <0.7.0;
+pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/math/SafeMath.sol";
```

Comment:
Why this change? 
is math "safe" by default in newer solidity versions?

Reply:
Hi. Yes, SafeMath is included from solidity version 0.8.4..